public_html/wp-content/plugins/wordcamp-speaker-feedback/includes/comment.php

```diff
@@ -74,6 +75,11 @@ function count_feedback( $post_id = 0 ) {
 				$feedback_count['total_comments'] += $row['total'];
 				$feedback_count['all']            += $row['total'];
 				break;
+			case 'inappropriate':
+				$feedback_count['inappropriate']   = $row['total'];
+				$feedback_count['total_comments'] += $row['total'];
+				$feedback_count['all']            += $row['total'];
```

Comment:
If we remove inappropriates from the "all" tab, you can also remove this count here.

Reply:
Updated in 53853129

Reply:
Addressed in 5385312